### PR TITLE
fix: switch to http for prometheus and grafana

### PR DESCRIPTION
**Pull request by burnerlee**
Currently using https for prometheus and grafana in our nginx-ingress config throws SSL Handshake error. This PR fixes this to host the services flawlessly

### CODE REVIEW

File: docker-compose-staging.yml [hunk 1]
```diff
@@ -130,7 +130,8 @@ services:
       - '--log.level=info'
       - '--web.telemetry-path=/statsd/metrics'
     ports:
-      - '9125:9125'
+      - '9125:9125/udp'
+      - '9125:9125/tcp'
```
Comment (Ram81) [inline docker-compose-staging.yml:134]: @burnerlee why is this needed?

**Review by Ram81 — CHANGES_REQUESTED** [review]

File: docker-compose-staging.yml (hunk 1 above)
Comment (burnerlee) [reply docker-compose-staging.yml:134]: @Ram81 by default, docker only exposes the tcp port, while the statsd-exporter listens to both the tcp and udp port. So I have to explicity mention both of them for both of them to be exposed

**Comment by burnerlee** [comment]
One file was pushed earlier by mistake. This PR removes that file as well

**Review by Ram81 — APPROVED** [review]
LGTM

**Review by RishabhJain2018 — APPROVED** [review]
